src/util/formFields.tsx

```diff
@@ -28,3 +28,6 @@ export const optionRenderer = (
 
   return "";
 };
+
+export const getTextareaRows = (textLength?: number) =>
```

Comment:
This is very opinionated about the width of the textarea, which might be variable -- especially due to responsiveness.

Maybe introduce a new component `DynamicTextarea` (or find a better name), that wraps the Textarea. In it, we can use a ref to measure the actual width of the textarea and move `getTextareaRows`  into this component, to have the dynamic row height adjusting to the textarea width.

If it works well, we can later upstream it to react-components original `Textarea` with a boolean prop to opt into the row dynamics.

Reply:
Probably out of scope, let's consider this as a future improvement.